postgres/pgdump.go

```diff
@@ -249,6 +249,13 @@ func processCreateStmt(conv *internal.Conv, n nodes.CreateStmt) {
 		logStmtError(conv, n, fmt.Errorf("can't get table name: %w", err))
 		return
 	}
+	if len(n.InhRelations.Items) > 0 {
+		// We currently skip the inherited tables in processing. so we log this
+		// behaviour in verbose mode, and skip the statement.
+		conv.SkipStatement(prNodes([]nodes.Node{n}))
```

Comment:
I think it is work adding an explicit 'unexpected' message like:

   		conv.Unexpected(fmt.Sprintf("Found inherited table %s -- we do not currently handle inherited tables", table))

Reply:
added unexpected message